### ISSUE TITLE
Unknown column type: set

### ISSUE THREAD

**Issue by arthurchumak** [Kononnable/typeorm-model-generator#91]
Mysql table column has datatype "SET('foo', 'bar')"

```
Unknown column type: set  table name: user column name: role
Error occured in typeorm-model-generator.
typeorm-model-generator@0.2.18  node@v6.13.0
If you think this is a bug please open an issue including this log on https://github.com/Kononnable/typeorm-model-generator/issues
Error
    at Object.LogError (/Users/ac/Projects/eschool/node_modules/typeorm-model-generator/dist/src/Utils.js:10:21)
    at response.filter.forEach.resp (/Users/ac/Projects/eschool/node_modules/typeorm-model-generator/dist/src/drivers/MysqlDriver.js:160:39)
    at Array.forEach (native)
    at entities.forEach.ent (/Users/ac/Projects/eschool/node_modules/typeorm-model-generator/dist/src/drivers/MysqlDriver.js:38:22)
    at Array.forEach (native)
    at MysqlDriver.<anonymous> (/Users/ac/Projects/eschool/node_modules/typeorm-model-generator/dist/src/drivers/MysqlDriver.js:33:22)
    at next (native)
    at fulfilled (/Users/ac/Projects/eschool/node_modules/typeorm-model-generator/dist/src/drivers/MysqlDriver.js:4:58)
    at process._tickCallback (internal/process/next_tick.js:109:7)
[2:44:31 PM] Typeorm model classes created.
```

**Comment by Kononnable** [comment]
I don't see support for set column in typeorm([`column types`](https://github.com/typeorm/typeorm/blob/master/docs/entities.md#column-types-for-mysql--mariadb)).
Column must be  supported in typeorm before typeorm-model-generator - otherwise generator would generate broken entity models.

**Comment by CatsMiaow** [comment]
```sql
`daily_days` set('sun','mon','tue','wed','thu','fri','sat') NOT NULL DEFAULT ''
```
<img width="211" alt="2018-12-14 2 53 19" src="https://user-images.githubusercontent.com/1300172/49985645-0c455f00-ffb0-11e8-8d2c-a29d52965408.png">

[sequelizejs](http://docs.sequelizejs.com/manual/tutorial/models-definition.html#data-types) does not support SET, but [sequelize-auto](https://github.com/sequelize/sequelize-auto) is treated as a string.
<img width="527" alt="2018-12-14 2 53 28" src="https://user-images.githubusercontent.com/1300172/49985653-10717c80-ffb0-11e8-9f70-d2865c020948.png">

I think, we would like to treat it as a string and issue a warning so that no errors occur.

**Comment by CatsMiaow** [comment]
https://github.com/typeorm/typeorm/issues/2779
https://github.com/typeorm/typeorm/pull/4538
https://github.com/typeorm/typeorm/blob/master/CHANGELOG.md#features

**Comment by Maia-Everett** [comment]
As a workaround, for now I've patched my local copy of the generator to map the `set` type to strings, and I join/split the sets manually.

**Comment by Kononnable** [comment]
Implemented in `typeorm-model-generator@0.4.0`
